roles/redis/README.md

```
@@ -0,0 +1,19 @@
+This role installes Redis in a docker container and configures it for the use with artemis.
+
+Please install docker before continuing with this role.
+
+Role Variables
+--------------
+
+Default variables can be found in the `defaults/main.yml` file.
+
+You have to configure the follwoing varaibles in your ansible `group_vars`:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)